### ISSUE TITLE
Synchronization failing on v2.1.3 (F-Droid ver.)

### ISSUE THREAD

**Issue by digital-mystik** [stefan-niedermann/nextcloud-notes#650]
**Android version:** 9 (One UI version 1.0)

**Device**: SM-G955U

**System language**: English (US)

**App version:** 2.1.3

**App source:** F-Droid

**Steps to reproduce:**
  1. Started from scratch by removing all android devices and sessions from Nextcloud instance via web UI, and unistalled all applications relating to it (client, davx5, nextcloud-notes)
  2. Installed Nextcloud client (F-Droid ver. 3.9.2) and logged in with 2FA enabled
  3. Installed Nextcloud Notes and selected account from "choose an account" prompt
  4. Allowed Notes to access account
  5. `Synchronization failed: is the Notes app activated on the server?` appears on refresh

Sidebar shows that my account is logged in, but does not show up in the web UI `Devices & sessions` (not sure if it is supposed to)

I believe the issues began with v2.0+ for me.

Thank you for your time.

**Comment by stefan-niedermann** [comment]
sounds like a dumb question, but: you **did** install the notes app on your server too, right? https://apps.nextcloud.com/apps/notes

What happens if you add a new note via the WebUI and synchronize then?

Can you provide a logcat (e. g. via aLogcat or another logcat app)?

**Comment by digital-mystik** [comment]
> sounds like a dumb question, but: you **did** install the notes app on your server too, right? https://apps.nextcloud.com/apps/notes
> 
> What happens if you add a new note via the WebUI and synchronize then?
> 
> Can you provide a logcat (e. g. via aLogcat or another logcat app)?

The notes app is installed on the instance by my provider. I am able to update and add notes via a synced desktop folder that I have configured on my PC, and the web UI picks up the changes so they do work (this syncing goes both ways). I have used the android notes client in the past without issue.

Here is the logcat output from what I could gather: https://gist.github.com/digital-mystik/aff1d206aaa8683b045f4e035acfeb03

I may have accidentally replaced some information like IDs and hashes that is necessary in the sake of privacy; I'm a bit of a noobie when it comes to text.
